styles/colors-silicon-day.less

```diff
@@ -183,6 +183,16 @@
 @agate-item-selected-border-color:        @agate-accent;
 @agate-item-selected-color:               @agate-item-label-color;
 
+// Keypad
+// ---------------------------------------
+@agate-keypad-button-active-bg-color: 		transparent;
+@agate-keypad-button-active-border-color: 	@agate-accent;
+@agate-keypad-button-active-color: 			@agate-keypad-button-active-border-color;
+@agate-keypad-button-bg-color: 				transparent;
+@agate-keypad-button-color: 				@agate-foreground;
+@agate-keypad-button-focus-color: 			@agate-white;
+@agate-keypad-button-sub-text-color: 		@silicon-dark-gray;
```

Comment:
I've we're going to align these values, we (sadly) must use spaces so the different tab-widths don't throw off the alignment. :)